source/user-manual/ruleset/ruleset-xml-syntax/decoders.rst

```diff
@@ -213,7 +215,7 @@ Define that the decoder is related with the ``test``, ``TEST`` or equivalent (ca
 prematch
 ^^^^^^^^^
 
-It attempts to find a match within the log for the string defined; if it finds a match, the current decoder will be used, the search for a decoder will stop, and only its child decoders will be able to match. It is important to be as specific as possible to avoid matching with wrong events.
+It defines a regular expression that the log must match to apply the decoder. Only the log after any syslog-like header is used for the match finding. It's important to be as specific as possible to avoid matching with wrong events.

```

Comment:
```suggestion
Defines a regular expression that the log must match to apply the decoder.  It's important to be as specific as possible to avoid matching unwanted events. Note that if the log is Syslog-like, then ``prematch`` only analyzes the log after the Syslog-like header. If the log is not Syslog-like, then it analyzes the entire log.
```